doc/usage.rst

```diff
@@ -138,6 +138,39 @@ Alternative hashing algorithms supported by :mod:`hashlib` can be used if necess
     print(pooch.file_hash("data/c137.csv", alg="sha512"))
 
 
+Bypassing the hash check
+------------------------
+
+Sometimes we might not know the hash of the file or it could change on the server
+periodically. In these cases, we need a way of bypassing the hash check.
+One way of doing that is with Python's ``unittest.mock`` module, and setting the
+hash value to ``ANY`` (a special object which passes all equality tests).
+
+In this example, we are downloading a list of weather stations around Australia:
```

Comment:
```suggestion
In this example, we want to use Pooch to download a list of weather stations around 
Australia. The file with the stations is in an FTP server and we want to store it locally 
in separate folders for each day that the code is run. The problem is that the 
``stations.zip`` file is updated on the server instead of creating a new one, so the 
hash check would fail. This is how you can solve this problem:

```